gear-programs/bridging-payment-vara-supply/src/services/utils.rs

```diff
@@ -0,0 +1,32 @@
+use super::{
+    error::Error, vft_treasury::vft_treasury::io as vft_treasury_io,
+    vft_treasury::Error as VftTreasuryError,
+};
+use sails_rs::calls::ActionIo;
+use sails_rs::prelude::*;
+
+pub async fn send_message_with_gas_for_reply(
```

Comment:
Maybe also encode message here? anyway it depends on the message format of `vft-treasury` and as I see `send_message_with_gas_for_reply` is called only inside `send_message_to_treasury`

Reply:
Extracted encoding/decoding  to `send_message_to_treasury`